src/main/kotlin/no/nav/syfo/application/api/authentication/ApiFeature.kt

```diff
@@ -66,6 +69,10 @@ fun ApplicationCall.personIdent(): PersonIdentNumber? {
     return principal?.payload?.subject?.let { PersonIdentNumber(it) }
 }
 
+fun ApplicationCall.personIdentAT(): PersonIdentNumber? {
+    return this.request.headers["personIdentAT"]?.let { PersonIdentNumber(it) }
+}
```

Comment:
Denne trengs ikke.

Reply:
Men vi trenger vel fnr for både Arbeidstaker og Nærmeste Leder?

Vi bruker personIdentAT for å hente Nærmeste Ledere til den sykmeldte, for å så filtrere disse på fnr til Nærmeste Leder.

Reply:
Har du sett kommentaren min nedenfor?